doc/MANUAL.adoc

```diff
@@ -1160,6 +1161,8 @@ Examples:
 
 * `+redis://localhost+`
 * `+redis://p4ssw0rd@cache.example.com:6379/0|connect-timeout=50+`
+* `+redis+unix:///run/redis.sock+`
```

Comment:
It would be nice to support `redis+unix:/run/redis.sock` as well, similar to the `file` schema.

Reply:
Sounds reasonable, I think it is "no hostname" vs "empty hostname" (or somesuch)